guides/03_features/03_padrino-mailer.html.md

```diff
@@ -102,7 +110,7 @@ MyAppName.mailer :sample do
     locals :name => name, :email => email
     render 'sample/registration_email'
     content_type :html # optional, defaults to :plain
-    via :sendmail      # optional, to smtp if defined otherwise sendmail
+    via :sendmail, settings     # optional, to smtp if defined otherwise sendmail
```

Comment:
I feel like instead of `settings`, we should provide a real value here to prevent confusion.

```ruby
via :sendmail, :location => "/usr/bin/sendmail"
```

Or perhaps leave the optional `settings` value out of this block, and expand the Configuration section above to include possible settings and where in the Mail gem the documentation can be found.

Thoughts?